### DISCUSSION TITLE
Can't close window when use ttkbootstrap

### DISCUSSION THREAD

**Discussion by TooLazy0x00** [israel-dryer/ttkbootstrap#273] [Q&A] [answered]
Can't close window when use ttkbootstrap

Great library, I found a small problem in the process of using it.

My python version is 3.7.9, and I use ttkbootstrap to build a project. This happens in IDLE，when the project is running, it cannot be closed by the functions root.quit() & root.destroy() or by clicking the x in the upper right corner of the window.

But when I use  ttk directly, this problem doesn't appear.

How to close such a window?

**Comment by israel-dryer** [comment, marked as answer]
This is definitely a bug. Thanks for pointing it out. I just pushed a fix in 1.7.6 that should take care of this issue #275

**Reply by TooLazy0x00** [reply]
Wow, everything worked fine after the upgrade. Really fast!
Hope this library gets better and better~~~
